### PR TITLE
Add face refresh function

### PR DESCRIPTION
**Pull request by thblt**
Add an interactive function that recomputes all colors, for eg theme changes.

### CODE REVIEW

File: erc-hl-nicks.el [hunk 1]
```diff
@@ -344,6 +344,13 @@
                   (erc-button-add-face (car bounds) (cdr bounds)
                                        (erc-hl-nicks-make-face trimmed)))))))))))
 
+(defun thblt/erc-hl-nicks-refresh-colors ()
```
Comment (leathekd) [inline erc-hl-nicks.el]: Could you remove the `thblt/` prefix?

Comment (thblt) [reply erc-hl-nicks.el]: Err, sorry for the mistake, this comes from my init.el. Should be good now.

Comment (leathekd) [reply erc-hl-nicks.el]: Oh, no worries!  Thanks for the PR.

**Comment by leathekd** [comment]
Thanks for contributing!  

Melpa will eventually rebuild (I think it's daily?).  I've updated the version and changelog and also pushed a new tag (1.3.4) in case you use Melpa Stable. 🚀

**Pull request merged**